File: app/src/main/java/com/blues/database/course/Course.java
package com.blues.database.course;


import androidx.room.ColumnInfo;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity(tableName = "course")
public class Course {

    @PrimaryKey(autoGenerate = true)
    private int id;

    @ColumnInfo(name = "name")
    private String name;

    @ColumnInfo(name = "teacher")
    private String teacher;

    @ColumnInfo(name = "classroom")
    private String classroom;

    @ColumnInfo(name = "st_num")
    private String st_num;

    @ColumnInfo(name = "class_name")
    private String class_name;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getTeacher() {
        return teacher;
    }

    public void setTeacher(String teacher) {
        this.teacher = teacher;
    }

    public String getClassroom() {
        return classroom;
    }

    public void setClassroom(String classroom) {
        this.classroom = classroom;
    }

    public String getSt_num() {
        return st_num;
    }

    public void setSt_num(String st_num) {
        this.st_num = st_num;
    }

    public String getClass_name() {
        return class_name;
    }

    public void setClass_name(String class_name) {
        this.class_name = class_name;
    }
}

File: app/src/main/java/com/blues/course/model/Course.kt
package com.blues.course.model

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "course")
data class Course(
        @PrimaryKey
        @ColumnInfo(name = "name")
        val courseName: String,
        @ColumnInfo(name = "teacher")
        val teacherName: String,
        @ColumnInfo(name = "classroom")
        val classroomName: String,
        @ColumnInfo(name = "st_num")
        val className: String,
        @ColumnInfo(name = "class_name")
        val studentNum: Int)